### PR TITLE
(kubelogin) add package for kubelogin cli

### LINKED ISSUE #1511: (kubelogin) adding a new automatic package?
I would like to contribute an automatic package for kubelogin.
The maintainer is kind enough to offer 64 bit builds on github releases: https://github.com/int128/kubelogin/releases

Before starting the work, I would ask if this something your open to or should I maintain the chocolatey package in my own repo.

### PR DESCRIPTION
**Pull request by jetersen**
## Description
Added a new package 😅
fixes #1511 

## Motivation and Context
`choco install kubelogin`

## How Has this Been Tested?
```powershell
> choco install -s . kubelogin

> Get-Command kubelogin

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     kubelogin.exe                                      0.8.1.0    C:\ProgramData\chocolatey\bin\kubelogin.exe

> kubelogin --version
kubelogin version v1.20.0
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by jetersen** [comment]
Is there anything I missed or should I go ahead and upload the initial package to get moderation going @AdmiringWorm ?

**Comment by RedBaron2** [comment]
@jetersen 
Might I make the suggestion to have the nuspec version be `0.0` or `0.0.0`
The AU script will change the version to the correct version found in the update script.

File: automatic/kubelogin/kubelogin.nuspec [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubelogin</id>
+    <version>1.20.0</version>
```
Comment (jetersen) [inline automatic/kubelogin/kubelogin.nuspec:6]:
```suggestion
    <version>0.0.0</version>
```

Comment (jetersen) [reply automatic/kubelogin/kubelogin.nuspec]: @RedBaron2 done using a github suggestion 👍

**Comment by majkinetor** [comment]
LGTM.

**Pull request merged**